syz-cluster/pkg/db/spanner.go

```diff
@@ -104,24 +104,29 @@ func dropSpannerDB(ctx context.Context, uri ParsedURI) error {
 //go:embed migrations/*.sql
 var migrationsFs embed.FS
 
-func RunMigrations(ctx context.Context, uri string) error {
-	sourceDriver, err := iofs.New(migrationsFs, "migrations")
+func RunMigrations(_ context.Context, uri string) error {
```

Comment:
`_ context.Context` looks unusual. Why is it needed?

Reply:
Dropped